templates/generate_template.sh

```diff
@@ -30,7 +30,7 @@ EOH
 }
 
 
-while getopts "t:g:w:f:r:h" opt;
+while getopts "t:g:w:f:r:h:" opt;
```

Comment:
`h:`

I don't think there's any reason to require an argument here. The argument would be unused, and if you do use the -h option without an argument, the rest of the script will run unintentionally, since the exit is not called.